apps/o2-blocks/index.php

```diff
@@ -1,40 +1,72 @@
 <?php
-/*
+/**
  * Plugin Name: A8C Blocks
  * Description: A8C Specific Blocks
  * Author: A8C
+ * Text Domain: full-site-editing
+ *
+ * @package o2-blocks
  *
  * Instructions:
- * 1. Build `o2-blocks`: `yarn build`
- * 2. Start a test WP instance at the root of the `o2-blocks` repo: `wp-env start`
- * 3. When you access your test WP instance, you should then see the `A8C Blocks` plugin installed and activated.
+ *   1. The block code is found in Calypso in `apps/o2-blocks` - see the README there for
+ *      information on how to edit and maintain them.
+ *      https://github.com/Automattic/wp-calypso/tree/trunk/apps/o2-blocks
+ *
+ *   2. The block code is built from a TeamCity job in Calypso
+ *      https://github.com/Automattic/wp-calypso/blob/a13e6ddacdf59ce5f2d6b26c22900c0f4afd1adb/.teamcity/_self/projects/WPComPlugins.kt#L107
+ *
+ *      It's built on every Calypso deploy. Why not just the ones where we modify o2-blocks?
+ *      Changes to Calypso's components, build system, and framework may indirectly change
+ *      the build of these blocks. We expect to update code here on every change to the
+ *      o2-blocks code but it's safe and normal from time to time to go ahead and rebuild the
+ *      blocks to capture ancillary work going on in general.
+ *
+ *   3. Login to your sandbox and fetch the updated block code with the install-plugins.sh script
+ *
+ *      ```
+ *      # Prep the latest trunk build for release
+ *      wpdev~/public_html# install-plugin.sh o2-blocks --release
+ *
+ *      # Alternatively, load changes from a branch (e.g. to test a PR.)
+ *      ```
+ *      wpdev~/public_html# install-plugin.sh o2-blocks $brach_name
+ *      ```
  */
 
+/**
+ * Load editor assets.
+ */
 function a8c_editor_assets() {
-	$assets = require( plugin_dir_path( __FILE__ ) . 'dist/editor.asset.php' );
+	$assets = require plugin_dir_path( __FILE__ ) . 'dist/editor.asset.php';
 
 	wp_enqueue_script(
 		'a8c-blocks-js',
 		plugins_url( 'dist/editor.js', __FILE__ ),
 		$assets['dependencies'],
-		$assets['version']
+		$assets['version'],
+		true
 	);
 
+	$style_file = 'dist/editor' . is_rtl() ? '.rtl.css' : '.css';
```

Comment:
Should this expression be:

$style_file = 'dist/editor' . (is_rtl() ? '.rtl.css' : '.css');

I guess PHP would always evaluate it to true due to the string being concatenated with `is_rtl()`'s value?